vspace/src/main/java/edu/asu/diging/vspace/web/CustomResponses.java

```diff
@@ -0,0 +1,20 @@
+package edu.asu.diging.vspace.web;
+
+public enum CustomResponses {
```

Comment:
unnecessary (see above), but if it would  be needed, this is a bad class (enum) name.